source/blog/2020-04-26-release-notes.md

```diff
@@ -0,0 +1,114 @@
+---
+layout: post
+title: April 2020 Patch Release Notes - Version 20.4.1
+published: true
+post_author:
+  display_name: Yufan Gong
+  twitter: yufangong
+tags: Releases, Finagle, Finatra, Scrooge, TwitterServer, Util
+---
+
+April Release encountered some issues running on JDK 8, here's a patch release hopefully fixes it. 
```

Comment:
"here's a patch release hopefully fixes it." --> "so here's a patch release [that should address it]." ?